TestApplication/SUTestApplicationDelegate.m

```diff
@@ -0,0 +1,184 @@
+//
+//  SUTestApplicationDelegate.m
+//  Sparkle
+//
+//  Created by Mayur Pawashe on 7/25/15.
+//  Copyright (c) 2015 Sparkle Project. All rights reserved.
+//
+
+#import "SUTestApplicationDelegate.h"
+#import "SUUpdateSettingsWindowController.h"
+
+@interface SUTestApplicationDelegate ()
+
+@property (nonatomic) SUUpdateSettingsWindowController *updateSettingsWindowController;
+@property (nonatomic) NSTask *serverTask;
+
+@end
+
+@implementation SUTestApplicationDelegate
+
+#define UPDATED_VERSION @"2.0"
```

Comment:
Make this a static const var unless there's actually a reason to make it a define (like overriding in build settings).